src/sysusers/sysusers.c

```diff
@@ -1525,22 +1525,22 @@ static int parse_line(const char *fname, unsigned line, const char *buffer) {
         r = extract_many_words(&p, NULL, EXTRACT_UNQUOTE,
                                &action, &name, &id, &description, &home, &shell, NULL);
         if (r < 0)
-                return log_error_errno(r, "[%s:%u] Syntax error.", fname, line);
+                return log_error_errno(r, "%s:%u: Syntax error.", fname, line);
```

Comment:
maybe switch to log_syntax() anyway? would mean clickable links and stuff...

Reply:
Hmm, I don't think log_syntax does clickable links. But it's better to use it anyway, I changed the patch now.

Reply:
if not, then we should add it ;-)